### DISCUSSION TITLE
Weird memory error

### DISCUSSION THREAD

**Discussion by sabracrolleton** [rabbibotton/clog#256] [Q&A] [answered]
Using sbcl

If I have loaded both trivial-ssh and cl+ssl, I can connect to a remote postgresql database from the repl using cl+ssl. (Trivial-ssh is going to be used for another purpose.) 

If I try to do the same thing in a clog application, trying to make the connection to the database triggers a memory fault error. The error message on the memory location seems to be consistent  

If I have not loaded trivial-ssh, no problem using cl+ssl in the clog application to connect to the postgresql database.

Anyone have any suggestions on how I should go about debugging this?

**Comment by sabracrolleton** [comment, marked as answer]
Never mind. Problem not in clog.

**Reply by rabbibotton** [reply]
Glad I could help! (Sorry have been had a very busy one and a half with happy things)
